core/store/orm/schema.go

```diff
@@ -35,7 +35,7 @@ type ConfigSchema struct {
 	ExplorerURL               *url.URL       `env:"EXPLORER_URL"`
 	ExplorerAccessKey         string         `env:"EXPLORER_ACCESS_KEY"`
 	ExplorerSecret            string         `env:"EXPLORER_SECRET"`
-	LogLevel                  LogLevel       `env:"LOG_LEVEL" default:"info"`
+	LogLevel                  LogLevel       `env:"LOG_LEVEL" default:"debug"`
```

Comment:
I think we want the default level to remain info.

Reply:
This was mistakenly left in